tests/integration/examples_test.py

```diff
@@ -49,3 +50,23 @@ def test_add_remove_linux_bridge(eth1_up):
         assertlib.assert_state(desired_state)
 
     assertlib.assert_absent('linux-br0')
+
+
+def test_add_remove_routes(eth1_up):
+    """
+    Test adding, downing and removing a vlan
+    """
+    route = {
+        Route.DESTINATION: '192.0.3.0/24',
+        Route.METRIC: '150',
+        Route.NEXT_HOP_ADDRESS: '192.0.2.1',
+        Route.NEXT_HOP_INTERFACE: 'eth1',
+        Route.TABLE_ID: '254'
+    }
+    with example_state('eth1_add_route.yml',
+                       cleanup='eth1_del_route.yml') as desired_state:
+        assertlib.assert_state(desired_state)
+        with example_state('eth1_del_route.yml') as desired_state:
+            assertlib.assert_route_absent(route)
```

Comment:
Since the example removes all routes, the assertion should IMHO assert that there are no routes for eth1 left in config. Otherwise it would not catch if for example for some reason the metric is just changed slightly.